ocean_lib/data_provider/data_service_provider.py

```diff
@@ -57,7 +57,7 @@ def set_http_client(http_client: Session) -> None:
     @staticmethod
     @enforce_types
     def encrypt(
-        objects_to_encrypt: Union[list, str, bytes], encrypt_endpoint: str
+        objects_to_encrypt: Union[list, str, bytes], provider_uri: str
```

Comment:
please make it like for other functions: optional. If empty, grab the one from config.

Reply:
Sure